sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -37,6 +37,28 @@ import { UserAgentOptions } from '@azure/core-http';
 import { UserDelegationKeyModel } from '@azure/storage-blob';
 import { WebResource } from '@azure/core-http';
 
+// @public
+export interface AccessControlChangeCounters {
+    changedDirectoriesCount: number;
+    changedFilesCount: number;
+    failedChangesCount: number;
+}
+
+// @public
+export interface AccessControlChangeFailure {
+    errorMessage: string;
+    isDirectory: boolean;
+    name: string;
+}
+
+// @public
+export interface AccessControlChanges {
+    aggregateCounters: AccessControlChangeCounters;
+    batchCounters: AccessControlChangeCounters;
+    batchFailures: AccessControlChangeFailure[];
+    continuation?: string;
```

Comment:
`continuation` shows up a lot - should this be `continuationKey` or `continuationToken` or something? I assume it's used to continue a previous operation?

Reply:
Should be `continuationToken` which aligns with others